### PR TITLE
Fix CI job running twice

### PR DESCRIPTION
**Pull request by pankajastro**
# Description
## What is the new behavior?

-
-
-

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: .github/workflows/ci-sql-cli.yaml [hunk 1]
```diff
@@ -21,15 +21,8 @@ defaults:
   run:
     working-directory: sql-cli
 jobs:
-  Markdown-link-check:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: gaurav-nelson/github-action-markdown-link-check@v1
-        with:
-          config-file: '.github/workflows/mlc_config.json'
```
Comment (kaxil) [inline .github/workflows/ci-sql-cli.yaml:30]: We didn't need it here, we already run it project wide in Python SDK workflow

File: python-sdk/src/astro/settings.py [hunk 2]
```diff
@@ -37,6 +37,7 @@
     section="astro_sdk", key="run_raw_sql_response_size", fallback=-1
 )
 
+# Temp changes
```
Comment (kaxil) [inline python-sdk/src/astro/settings.py:40]: We should remove this line :(

File: .github/workflows/ci-python-sdk.yaml [hunk 3]
```diff
@@ -23,6 +23,7 @@ defaults:
     working-directory: python-sdk
 jobs:
   Markdown-link-check:
+    if: github.event.action != 'labeled'
```
Comment (kaxil) [inline .github/workflows/ci-python-sdk.yaml:26]: We don't want to run this jobs when a label is added as it is redundant with when PR is created

**Comment by pankajastro** [comment]
Thank you so much @kaxil for looking into it

**Pull request merged**